packages/esm-patient-test-results-app/src/trendline/trendline.component.tsx

```diff
@@ -113,7 +123,8 @@ const Trendline: React.FC<{
     interpretation?: OBSERVATION_INTERPRETATION;
   }> = [];
 
-  let dataset = patientData[0];
+  const dataset = patientData[0];
+  const referenceRange = patientData[1][0]?.meta?.range;
```

Comment:
It would be sweet if we could refactor `useTrendlineData` to return an object rather than an array, so people don't have to look up the implementation to figure out what these numeric indexes are supposed to represent.